package.json

```diff
@@ -37,29 +37,32 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@pixi/constants": "^5.0.0",
-    "@pixi/core": "^5.0.0",
-    "@pixi/display": "^5.0.0",
-    "@pixi/graphics": "^5.0.0",
-    "@pixi/math": "^5.0.0",
-    "@pixi/sprite": "^5.0.0"
+    "@pixi/constants": "^5.4.0-rc.3",
+    "@pixi/core": "^5.4.0-rc.3",
+    "@pixi/display": "^5.4.0-rc.3",
+    "@pixi/graphics": "^5.4.0-rc.3",
+    "@pixi/math": "^5.4.0-rc.3",
+    "@pixi/sprite": "^5.4.0-rc.3",
+    "@pixi/utils": "^5.4.0-rc.3"
   },
   "devDependencies": {
     "@microsoft/api-extractor": "^7.9.4",
-    "@pixi-build-tools/rollup-configurator": "^1.0.3",
-    "@pixi/canvas-renderer": "^5.3.3",
-    "@pixi/constants": "^5.3.3",
-    "@pixi/core": "^5.3.3",
-    "@pixi/display": "^5.3.3",
-    "@pixi/graphics": "^5.3.3",
-    "@pixi/math": "^5.3.3",
-    "@pixi/sprite": "^5.3.3",
+    "@pixi-build-tools/rollup-configurator": "~1.0.11",
+    "@pixi/canvas-renderer": "5.4.0-rc.3",
```

Comment:
Recommend ^6.0.0-X

Reply:
Cool, I guess the dependencies can have 5.4.0-X while devDependencies are v6. No breaking changes, _yet_!

Reply:
Huh? I don't understand, this is kinda confusing. 5.4.0 is not a thing and will never come out. Why not bump everything to 6?

Reply:
Ok, deps too to v6 :). I though some people might not want to do a major bump but I guess it doesn't matter.